theorems/T000171.md

```diff
@@ -3,9 +3,7 @@ uid: T000171
 if:
   P000035: true
 then:
-  and:
-  - P000003: true
-  - P000030: true
+  P000030: true
```

Comment:
This file in now a complete duplicate of T12 (and furthermore T12 points to a correct justification in S&S, whereas Fig 7 mentioned here does not seem to be it).  Delete file?

On the other hand, how do we now get [Fully T4 ==> T2] ?

Reply:
![image](https://user-images.githubusercontent.com/1559632/229149256-17f6f64f-d918-4558-898a-a5d2636e418e.png)